canonical-web-and-design/maas.io
# Update global-nav and add hiring option

## Done
Upgrade the global-nav module to include the hiring feature.
Provided the hiring option to display a link in the header

## QA
- Load the site
- See that the global-navigation does not display login but a hiring link which works

## Screenshots
![Screenshot_2019-11-15 Metal as a Service MAAS](https://user-images.githubusercontent.com/1413534/68882337-e1571400-0706-11ea-915b-7b03d4b9e0ed.png)
